### PR TITLE
Ri program2

### PR DESCRIPTION
**Pull request by ritheesh4**

### CODE REVIEW

File: 00002_arithmetic.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*2. Enter 2 numbers and display the result for the following operations (each operation should be a function)
+    add / multiply / divide / modulus / substract / greater than / less than / is equal.
+*/
+
+const readLine = require("readline-sync");
+const number1 = readLine.question("Enter first Number : ");
+const number2 = readLine.question("Enter the second Number: ");
+
+let addition = (number1, number2) => {
```
Comment (kuism) [inline 00002_arithmetic.js]: you should not define a function with let

File: 00002_arithmetic.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    let result = parseInt(number1) / parseInt(number2);
+    console.log("Division:", number1, "/", number2, "=", result);
+}
+
+let modulus = (number1, number2) => {
+    let result = parseInt(number1) % parseInt(number2);
+    console.log("Modulus:", number1, "%", number2, "=", result);
+}
+
+let numberCompare = (number1, number2) => {
```
Comment (kuism) [inline 00002_arithmetic.js]: Return values instead of printing it

File: 00002_arithmetic.js (hunk 1 above)
Comment (ritheesh4) [reply 00002_arithmetic.js]: Ok. Changed every function declaration into const.

File: 00002_arithmetic.js (hunk 2 above)
Comment (ritheesh4) [reply 00002_arithmetic.js]: Ok. Console removed in every function  and returned the results.

**Pull request merged**